Source/CarthageKit/Xcode.swift

```diff
@@ -355,6 +357,65 @@ private func mergeExecutables(_ executableURLs: [URL], _ outputURL: URL) -> Sign
 		.then(SignalProducer<(), CarthageError>.empty)
 }
 
+private func findSwiftHeaderFiles(fromExecutableURLs executableURLs: [URL]) -> [URL] {
+	return executableURLs.map { 
+		let executableName = $0.lastPathComponent
+		return $0.deletingLastPathComponent().appendingPathComponent("Headers/\(executableName)-Swift.h") 
+	}
+}
+
+private func findSwiftHeaderOutputURL(fromURL url: URL) -> URL {
```

Comment:
What do you think of renaming this function to `swiftHeaderURL(fromURL url: URL) -> URL`

Reply:
We should be able to use the existing `URL.swiftHeaderURL()` extension instead of adding this new func.

Reply:
Used `URL.swiftHeaderURL()`.